package.json

```diff
@@ -7,8 +7,10 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^13.0.0",
     "@testing-library/user-event": "^13.2.1",
```

Comment:
Let's move, please, these libraries to `devDependencies` since they are not required for prod build, they will just make it more heavy